src/web/app/components/logs-table/logs-table.component.scss

```diff
@@ -16,9 +16,36 @@
 }
 
 .logs-table {
-  table-layout: fixed;
+  display: flex;
+  flex-flow: column;
   width: 100%;
+  height: 100%;
   word-break: break-all;
+  max-height: 90vh;
+}
+
+.logs-table thead {
```

Comment:
Quick comment: All these definitions are how they are done in vanilla CSS. Make use of SCSS nested selectors in order to prevent repeating `.logs-table` (and possibly more stuffs) everywhere.

Reply:
Got it, not familiar with SCSS but will look it up and make changes.